spark/build.sbt

```diff
@@ -7,7 +7,8 @@ libraryDependencies ++= Seq(
     exclude("org.apache.hadoop", "hadoop-client"),
   "org.apache.spark" %% "spark-core" % Version.spark % "provided",
   "org.apache.hadoop" % "hadoop-client" % Version.hadoop % "provided",
-  "de.javakaffee" % "kryo-serializers" % "0.27",
+  "de.javakaffee" % "kryo-serializers" % "0.27" exclude("com.esotericsoftware.kryo", "kryo"),
```

Comment:
Can downgrade to `javakaffee 0.24` only after we decide what to do with `org.joda.time.Interval` (is it really necessary to use it)